pkg/streamingpromql/operators/aggregation.go

```diff
@@ -235,49 +242,58 @@ func (a *Aggregation) NextSeries(ctx context.Context) (types.InstantVectorSeries
 			}
 		}
 	}
+	return floatPointCount
+}
 
-	var floatPoints []promql.FPoint
-	var err error
-	if floatPointCount > 0 {
-		floatPoints, err = a.Pool.GetFPointSlice(floatPointCount)
+func (a *Aggregation) seriesIntoGroup(s types.InstantVectorSeriesData, seriesGroup *group, steps int, start int64, interval int64) (err error) {
```

Comment:
[nit] It annoys me that the method name doesn't contain a verb - how about `accumulateSeriesIntoGroup`?